src/features/utils/instrument-base.js

```diff
@@ -9,6 +10,8 @@ export class InstrumentBase extends FeatureBase {
       })
       this.hasAggregator = false
       this.auto = auto
+
+      if (auto) registerDrain(agentIdentifier, featureName)
```

Comment:
Auto is used in tests and other places to inform the class to/not to import the aggregator.  It is also used to inform the class whether a drain should be registered (to be fulfilled by the `agg` class)